include/ceed/ceed.h

```diff
@@ -216,6 +216,10 @@ CEED_EXTERN const CeedVector CEED_VECTOR_NONE;
 /// @ingroup CeedBasis
 CEED_EXTERN const CeedBasis CEED_BASIS_NONE;
 
+/// DEPRECATED Superceded by `CEED_BASIS_NONE`
+/// @ingroup CeedBasis
+CEED_EXTERN const CeedBasis CEED_BASIS_COLLOCATED;
```

Comment:
Not sure how to best demarcate this in the docs, or if we want to have it documented at all.

Reply:
I wouldn't have it in the docs